node/internal.ts

```diff
@@ -171,8 +171,27 @@ export function _setResourcePath(path: string): void {
 }
 
 /**
+ * Imports localized strings from json file.
+ * It can be used by common modules to load module specific loc strings
+ *
+ * @param     path      Full path to the json.
+ * @returns   void
+ */
+export function _importLocStrings(path: string): void {
```

Comment:
I agree with this.

" It's important in the function intellisense comments to indicate it overlays and that authors of re-usable modules should prefix their resource keys to avoid collisions (for instance all task lib resource keys are prefixed LIB_)"